src/controllers/action/create-one.js

```diff
@@ -0,0 +1,22 @@
+const HttpStatus = require('http-status-codes');
+const boom = require('boom');
+const logger = require('../../lib/logger');
+
+const {
+    Action,
+    User,
+} = require('../../models');
+
+module.exports = (req, res, next) => {
+    const action = new Action({
+        icon: req.body.icon,
+        userId: req.userData.userId,
+        color: req.body.color,
+        description: req.body.description,
+        type: req.body.type,
+    });
+
+    action.save()
+        .then(result => res.sendStatus(HttpStatus.CREATED))
```

Comment:
i think that you should return id - we need to pass it when delete activity for example ;)